css/mango-page.css

```diff
@@ -157,9 +157,23 @@ h1, h2, h3, h4, h5, h6 {
 /* Mango additions for exercises and warnings */
 
 .exercisebox {
-  outline: 5px dotted green;
+  padding: 1em 1em 1em 4em;
+  margin-bottom: 10px;
+  outline: 2px solid LightGrey;
+  position:relative
 }
 
+.exercisebox:before {
+  content: "\f044";
+  font-family: FontAwesome;
```

Comment:
For me it was fixedby changing this to:

```css
font-family: "Font Awesome 5 Free";
```

Reply:
![image](https://user-images.githubusercontent.com/5878305/60040054-098a9b80-96b0-11e9-8c36-8cf8735a20b3.png)

Reply:
On my mac I get the same problem, even when using `font-family: "Font Awesome 5 Free";`
![fa_exercise_icon](https://user-images.githubusercontent.com/3776224/60087595-3634c680-9735-11e9-925c-983d7dccfea7.png)

Using an SVG actually would be simpler I think. I will give this a go.